shared/message/strategy/WalkToFieldPosition.proto

```diff
@@ -33,4 +33,6 @@ import "Transform.proto";
 message WalkToFieldPosition {
     /// Transform from desired robot pose {d} to field {f} space
     iso3 Hfd = 1;
+    /// Whether to stop at the target position
+    bool dont_stop_at_target = 2;
```

Comment:
I feel like this should be `stop_at_target`? to make it easier to read and not have double negatives in the code